src/app/content/highlights/components/cardUtils.ts

```diff
@@ -9,13 +9,32 @@ import { assertDefined } from '../../../utils/assertions';
 import { cardMarginBottom } from '../constants';
 import { HighlightData } from '../types';
 
-export const getHighlightOffset = (container: HTMLElement | undefined, highlight: Highlight) => {
+export const getHighlightOffset = (
+  container: HTMLElement | undefined,
+  highlight: Highlight,
+  expandableAncestorsSelector?: string
+) => {
   if (!container || !highlight.range || !highlight.range.getBoundingClientRect) {
     return;
   }
 
   const {top, bottom } = highlight.range.getBoundingClientRect();
 
+  if (expandableAncestorsSelector) {
+    const highlightElement = highlight.elements[0] as HTMLElement;
+    const expandableAncestor = highlightElement && highlightElement.closest(expandableAncestorsSelector);
```

Comment:
@TomWoodward  @PiotrKozlowski 
Do we want to handle expandable containers nested in other expandable container?

Reply:
I think we have any so I thnk we don't care about them